src/autora/doc/pipelines/main.py

```diff
@@ -16,13 +16,24 @@
 logger = logging.getLogger(__name__)
 
 
-@app.command()
-def eval(data_file: str, model_path: str, sys_id: SystemPrompts, instruc_id: InstructionPrompts) -> List[str]:
+@app.command(help="Evaluate model on a data file")
+def eval(
+    data_file: str = typer.Argument(..., help="JSONL Data file to evaluate on"),
+    model_path: str = typer.Option("meta-llama/Llama-2-7b-chat-hf", help="Path to HF model"),
```

Comment:
Does this model already exist? If not, this might be better if it's a required argument?

Reply:
It does exist and will be downloaded on-demand from huggingface on first use if it's not already in the cache:

https://huggingface.co/meta-llama/Llama-2-7b-chat-hf

Reply:
Gotcha, alrighty. If it will exist in file by default, then this is okay. Thanks!